examples/with-glamorous/pages/index.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react'
-import { css } from 'glamor'
+import { style, rehydrate, css } from 'glamor'
 import glamorous from 'glamorous'
 
+// Adds server generated styles to glamor cache.
```

Comment:
You could probably put this in a global component like a layout and get away with it. I haven't given it a try though.

FWIW, I'd call this a "working example" more than a "best practices example". 😉